src/gui/pedalboard/cables/Cable.cpp

```diff
@@ -102,7 +102,16 @@ void Cable::paint (Graphics& g)
 
         endPortLocation = CableViewPortLocationHelper::getPortLocation ({ endEditor, connectionInfo.endPort, true }).toFloat();
         endColour = endEditor->getColour();
-        levelDB = connectionInfo.endProc->getInputLevelDB (connectionInfo.endPort);
+        auto updatedLevelDb = jlimit (floorDB, 0.0f, connectionInfo.endProc->getInputLevelDB (connectionInfo.endPort));
+        auto levelDifference = std::abs (updatedLevelDb) - std::abs (levelDB);
+        if (std::abs (levelDifference) > 2.0f)
```

Comment:
I tried playing around with some different thresholds here... the cable visualizations aren't super sensitive, so I think 2 dB is okay here, but we should probably test it out a bit more, to make sure it looks smooth.

We should also probably skip repainting if both `levelDB` and `updatedLevelDb` are below `floorDB`. I also wonder if `floorDB` is a little too low... like the input/output level meters use a floor of -45 dB. Idk, just more stuff to experiment with... and we can always tweak it later.

Reply:
Mmm, I'm a bit confused with this. Won't the `updatedLevelDb` and `levelDB` always be above the floorDB, because of the `jlimit (floorDB, 0.0f, levelDB)`?

Reply:
I implemented something similar to this in the latest changes. Let me know if you think it's efficient as it can be.